spacy_llm/tasks/span.py

```diff
@@ -143,6 +172,20 @@ def assign_spans(
         """Assign spans to the document."""
         raise NotImplementedError()
 
+    def _format_response(self, response: str) -> Iterable[Tuple[str, Iterable[str]]]:
+        """Parse raw string response into a structured format"""
+        output: dict[str, list[str]] = defaultdict(list)
```

Comment:
Let's do `Dict` and `List` so we don't have to do `from __future__ import annotation` (we support Python < 3.9 too).